src/diffusers/models/attention.py

```diff
@@ -245,7 +246,7 @@ def __init__(
         # project dropout
         self.net.append(nn.Dropout(dropout))
         # project out
-        self.net.append(nn.Linear(inner_dim, dim_out))
+        self.net.append(LinearWithLoRA(inner_dim, dim_out))
```

Comment:
Why is this being changed?

Reply:
Ah, this is why: https://github.com/huggingface/diffusers/pull/4147/files#diff-a89e72c24a390f4a4d50044cd7651a48b7e313411fa747bb430b1cf5e389bbf9R100